ludwig/schema/preprocessing.py

```diff
@@ -557,6 +557,40 @@ class AudioPreprocessingConfig(schema_utils.BaseMarshmallowConfig):
         "!= raw. The following parameters can/should be defined in the dictionary ",
     )
 
+    type: str = schema_utils.StringOptions(
```

Comment:
Review won't let me tag the correct line, but above in the definition for `missing_value_strategy` you could use the `BACKFILL` constant

Reply:
oh good call, will change!